sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/CustomEvents/ApplicationInsightsEventLogger.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.Logging;
+
+namespace Azure.Monitor.OpenTelemetry.Events
+{
+    /// <summary>
+    /// Default logger for logging Application Insights custom events.
+    /// </summary>
+    public sealed class ApplicationInsightsEventLogger : IApplicationInsightsEventLogger
+    {
+        private const string EventLoggerName = "Azure.Monitor.OpenTelemetry.CustomEvents";
+        private static readonly Func<IReadOnlyList<KeyValuePair<string, object?>>?, Exception?, string> s_formatter = (state, ex) =>
+        {
+            return "CustomEvent";
```

Comment:
use more detailed name, so folks using other log providers will not get confused seeing this..